server/data-import/data.js

```diff
@@ -31,7 +31,7 @@ async function reimport() {
 	await PregnancyCenterModel.collection.drop()
 
 	try {
-		const dataToImport = fs.readFileSync('server/data-import/test/fixtures/cessilye_nypc.json', 'utf8')
+		const dataToImport = fs.readFileSync('server/data-import/fixtures/cessilye_nypc.json', 'utf8')
```

Comment:
It looks like this change is still in this PR. Can you revert? You can run `git checkout origin/master server/data-import/data.js` and it will revert the file to the file that is on master.

Reply:
I did it but it still comparing it. Even the data_route PR has been merged but still showing the difference. not sure why?